lntest/itest/assertions.go

```diff
@@ -198,8 +198,11 @@ func closeChannelAndAssert(ctx context.Context, t *harnessTest,
 	net *lntest.NetworkHarness, node *lntest.HarnessNode,
 	fundingChanPoint *lnrpc.ChannelPoint, force bool) *chainhash.Hash {
 
+	ctxt, cancel := context.WithTimeout(ctx, defaultTimeout)
```

Comment:
Shouldn't we instead fix this in the test that doesn't pass in a context with timeout? I checked a few call sites of `closeChannelAndAssert()` and all that I looked at (except for the failing update channel policy for private channel test) pass in a context with `channelCloseTimeout` which is 4 times as long as the default timeout.

Reply:
How about we do this inside the function and remove all declarations of the two lines from the call sites?
```ctxt, cancel := context.WithTimeout(ctx, defaultTimeout) 
defer cancel()
```
I think it could save us quite a few lines, and remove the cumbersome of creating context with a timeout.

Reply:
Yes, I thought about that as well but saw that there are 84 call sites. If you're up to removing all of them I'd say go for it, yes!

Reply:
Actually loved it!